openshift/topics/tutorials.adoc

```diff
@@ -494,21 +506,22 @@ $ oc scale --replicas=1 dc/sso
 deploymentconfig "sso" scaled
 ----
 
-=== Example Workflow: Migrating The {project_name} Server's Database Across Environments
-This tutorial focuses on migrating the Red Hat Single Sign-On server database from one environment to another or migrating to a different database. It assumes steps described in xref:Preparing-SSO-Authentication-for-OpenShift-Deployment[Preparing {project_name} Authentication for OpenShift Deployment] section have been performed already.
+=== Migrating The {project_name} Server's Database Across Environments
+This tutorial focuses on migrating the Red Hat Single Sign-On server database from one environment to another or migrating to a different database.
+
+Export and import of {project_name} {project_version} database is triggered at {project_name} server boot time and its parameters are passed in via Java system properties. This means during one {project_name} server boot, only one of the possible migration actions, *_export_* or *_import_*, is performed.
 
 ==== Deploying the {project_name} PostgreSQL Application Template
 
+.Prerequisites
+* The steps described in the xref:Preparing-SSO-Authentication-for-OpenShift-Deployment[Preparing {project_name} Authentication for OpenShift Deployment] section have been performed already.
+
+.Procedure
 . Log in to the OpenShift web console and select the _sso-app-demo_ project space.
```

Comment:
Again, change `_sso-app-demo_` to `*sso-app-demo*` (looks we are using bold highlighted text below).